src/components/ComputeModal.js

```diff
@@ -737,7 +737,7 @@ export const ComputeModalBase = ({
       setCustomEnvImage(!foundImage ? imageUrl : '')
       setJupyterUserScriptUri(currentRuntimeDetails?.jupyterUserScriptUri || '')
 
-      const locationType = location === defaultLocation ? locationTypes.default : locationTypes.region
+      const locationType = getLocationType(location)
```

Comment:
The previous logic wasn't compatible with the change from `US` to `US-CENTRAL1`; specifically since the `location` variable was == to the `defaultLocation` (since the `defaultComputeLocation` is already `US-CENTRAL1`), we were setting the location type to `default` which is multi-region, which is incorrect.

I created a helper function in `region-common.js` that explicitly looks at the `allRegions` list and pulls the location type out that way rather than inferring it here.